js/pins.js

```diff
@@ -1,29 +1,43 @@
 'use strict';
 
 (function () {
-  var avatarNumbers = [1, 2, 3, 4, 5, 6, 7, 8];
+  var PinSize = {
```

Comment:
```suggestion
  var PIN_SIZE = {
```
https://up.htmlacademy.ru/javascript/17/criteries#b7